DQMOffline/Trigger/plugins/HTMonitor.cc

```diff
@@ -142,28 +162,24 @@ void HTMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup)
   reco::PFMET pfmet = metHandle->front();
   if ( ! metSelection_( pfmet ) ) return;
 
-  float ht = 0.0; 
-
-  edm::Handle<reco::PFJetCollection> jetHandle; //add a configurable jet collection & jet pt selection
+  edm::Handle<reco::JetView> jetHandle; //add a configurable jet collection & jet pt selection
   iEvent.getByToken( jetToken_, jetHandle );
-  std::vector<reco::PFJet> jets;
+  std::vector<reco::Jet> jets;
   if ( jetHandle->size() < njets_ ) return;
   for ( auto const & j : *jetHandle ) {
     if ( jetSelection_( j ) ) {
       jets.push_back(j);
     }
   }
-  for ( auto const & j : *jetHandle ) {
-    if ( jetSelection_HT_(j)) ht += j.pt();
-  }
+
   if ( jets.size() < njets_ ) return;
 
   float deltaPhi_met_j1 = 10.0;
   float deltaPhi_j1_j2 = 10.0;
 
-  if (!jets.empty()) deltaPhi_met_j1 = fabs( deltaPhi( pfmet.phi(),  jets[0].phi() ));
-  if (jets.size() >= 2) deltaPhi_j1_j2 = fabs( deltaPhi( jets[0].phi(),  jets[1].phi() ));
-  
+  if (!jets.empty()) deltaPhi_met_j1 = fabs( deltaPhi( pfmet.phi(),  jets.at(0).phi() ));
```

Comment:
Hi @marco-link 
[0] is better than at(0) here (you have already checked that the collection has >0 elements